cms/static/sass/course-unit-mfe-iframe-bundle.scss

```diff
@@ -0,0 +1,645 @@
+    }
+  }
+
+  textarea {
+    box-shadow: none !important;
+
+    @extend %input-styles;
+
+    &:hover {
+      border: solid 1px #002121;
+    }
+
+    &:focus {
+      color: $text-color !important;
+      background-color: $white;
+      border-color: $primary !important;
+      outline: 0;
+      box-shadow: 0 0 0 1px $primary !important;
+    }
+  }
+
+  .tip.setting-help {
+    color: $border-color;
+    font-size: 14px;
+    line-height: $base-font-size;
+  }
+
+  label,
+  .label.setting-label {
+    font-size: $base-font-size;
+    line-height: 28px;
+    color: $text-color;
+    font-weight: 400;
+  }
+
+  .title.modal-window-title {
+    color: $black;
+    font-weight: 700;
+    font-size: 22px;
+    line-height: 28px;
+  }
+
+  .xblock-actions {
+    background-color: $white !important;
+
+    .action-button {
+      background-color: $transparent;
+      border: 1px solid $transparent;
+      padding: 9px 16px;
+      border-radius: $input-border-radius;
+      font-weight: 400;
+      color: #00688d;
+      font-size: $base-font-size;
+      line-height: 20px;
+      cursor: pointer;
+
+      &:focus {
+        outline: none;
+      }
+    }
+
+    .openassessment_save_button,
+    .save-button,
+    .continue-button {
+      color: $white !important;
+      background-color: $primary;
+      border-color: $primary !important;
+      box-shadow: none;
+      font-weight: 500;
+      border: 1px solid $transparent !important;
+      padding: 10px 16px !important;
+      font-size: $base-font-size;
+      line-height: 20px;
+      border-radius: $btn-border-radius !important;
+      text-align: center;
+      vertical-align: middle;
+      user-select: none;
+      background-image: none;
+      height: auto !important;
+      display: block;
+
+      &:hover {
+        background: darken($primary, 5%);
+        border-color: $transparent;
+        color: $white;
+      }
+
+      &:focus {
+        background: $primary;
+        outline: none;
+      }
+    }
+
+    .openassessment_cancel_button,
+    .cancel-button {
+      color: $text-color;
+      background-color: $transparent;
+      border-color: $transparent;
+      box-shadow: none;
+      font-weight: 500;
+      border: 1px solid $transparent !important;
+      padding: 10px 16px !important;
+      font-size: $base-font-size;
+      line-height: 20px;
+      border-radius: $btn-border-radius !important;
+      text-align: center;
+      vertical-align: middle;
+      user-select: none;
+      background-image: none;
+      display: block;
+
+      &:hover {
+        background-color: $light-background-color;
+        border-color: $transparent;
+        color: $text-color;
+      }
+
+      &:focus {
+        background: $transparent;
+        outline: none;
+      }
+    }
+  }
+}
+
+body #openassessment-editor #openassessment_editor_header,
+body #openassessment-editor .openassessment_tab_instructions {
+  background-color: $white;
+}
```

Comment:
[nit] Let's include in the block below:
```
body {
    #openassessment-editor {
        #openassessment_editor_header,
        .openassessment_tab_instructions {
              background-color: $white;
        }
    }
...
}
```

Reply:
I did some additional refactoring
- removed everything `!important`
- removed access to elements via `body`